src/analysisd/alerts/exec.c

```diff
@@ -106,46 +107,145 @@ void OS_Exec(int execq, int *arq, const Eventinfo *lf, const active_response *ar
 
     /* Active Response to the forwarder */
     else if ((Config.ar & REMOTE_AR)) {
-        int rc;
-        /* If lf->location start with a ( was generated by remote agent and its
-         * ID is included in lf->location if missing then it must have been
-         * generated by the local analysisd, so prepend a false id tag */
-        if (lf->location[0] == '(') {
-            snprintf(exec_msg, OS_SIZE_1024,
-                     "%s %c%c%c %s %s %s %s %ld.%ld %d %s %s %s",
-                     lf->location,
-                     (ar->location & ALL_AGENTS) ? ALL_AGENTS_C : NONE_C,
-                     (ar->location & REMOTE_AGENT) ? REMOTE_AGENT_C : NONE_C,
-                     (ar->location & SPECIFIC_AGENT) ? SPECIFIC_AGENT_C : NONE_C,
-                     ar->agent_id != NULL ? ar->agent_id : "(null)",
-                     ar->name,
-                     user,
-                     ip,
-                     (long int)lf->time.tv_sec,
-                     __crt_ftell,
-                     lf->generated_rule->sigid,
-                     lf->location,
-                     filename ? filename : "-",
-                     extra_args ? extra_args : "-");
+
+        /* We need to identify the version of Wazuh installed on the agents because 
```

Comment:
The initial version for these changes is `4.2`. Please, change this to include version `4.2` in the new mechanism.

Reply:
Change accepted.

Reply:
I couldn't find the changes for this comment.

Reply:
It was my mistake, I recorded an old version over a new one.